docs/privacyidea.txt

```diff
@@ -136,6 +136,13 @@ Use the following example:
   // You must not change the class name
     'class'             => 'privacyidea:privacyidea',
 
+  // Other authproc filters can disable 2FA if you want to.
+  // For that you need to enter the path end the key, which should be true or false.
```

Comment:
Please be a bit more specific here in regards to "the path and the key".
Like the path and the key in the state object passed by the previous auth proc filter.